packages/sage/src/commands/export.ts

```diff
@@ -172,18 +177,29 @@ async function exportModule(module: string, project: ProjectInfo, commandOptions
 		logger.debug(`Missing dependencies:`, depResults.missing)
 		logger.info(`Installing missing dependencies...`)
 		try {
-			await exec(`${cmd(packageManager)} install ${missingDeps.join(' ')}`, {
-				cwd: exportPath
-			})
+			await exec(
+				`${cmd(packageManager == 'npm' ? 'npx' : packageManager)} ${await isUsingWorkspacesAndNpm(
```

Comment:
Forcefully using `npx` for `npm` doesn't always make sense here because it technically does still work when using npm workspaces. It should only be `npx` when using npm but not workspaces.

To keep things clean, maybe rename the `packageManager` variable to `command` and override it with `npx` when the criteria is met?